dockerfile

```diff
@@ -0,0 +1,22 @@
+# pull official base image
+FROM node:14.17-alpine
+
+# set working directory
+WORKDIR /parity-bridges-ui
+
+RUN apk update && \
+    apk add git
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /parity-bridges-ui/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY yarn.lock ./
+RUN yarn
+
+# add app
+COPY . ./
+
+# start app
+CMD ["npm", "start"]    
```

Comment:
This will start a development version, we rather want a production version, where we:
1. First build the static files.
2. Serve the static files using nginx.

Reply:
> 2. Serve the static files using nginx.

Will the nginx be included inside the bridges-ui docker or someone should run it as an external docker conatiner?